interface/usergroup/user_info.php

```diff
@@ -83,5 +122,103 @@ function(data)
 </HTML>
 <?php
   //  da39a3ee5e6b4b0d3255bfef95601890afd80709 == blank
+
+if (isset($_FILES)) {
+  //images will be saved with their uid
+  $uid = $_SESSION['authId'];
+    //MAKE THE UPLOAD DIRECTORY IF IT DOESN'T EXIST
+  if (realpath("../../profile_pictures/")) {
```

Comment:
755 permission if linux...

Reply:
windows doesnt need permissions it seems, are you working in windows?
i am using ubuntu it tests well here.

Reply:
Good!